app/services/testdata/OnlineTestInvitedStatusGenerator.scala

```diff
@@ -43,7 +43,8 @@ trait OnlineTestInvitedStatusGenerator extends ConstructiveGenerator {
       onlineTestUrl = generatorConfig.cubiksUrl,
       invitationDate = DateTime.now(),
       expirationDate = DateTime.now().plusDays(7),
-      participantScheduleId = 149245
+      participantScheduleId = 149245,
```

Comment:
Last time, in fasttrack, we added "participantScheduleId" because it was returned by cubiks but finally we did not use it but we keep it because of time constraints. It might be worth to revisit this and remove it if it is not necessary.